### ISSUE TITLE
Zones are not removed when removed from HA

### ISSUE THREAD

**Issue by sgruby** [home-assistant/home-assistant-iOS#224]
I added a zone in HA and it showed up on the iOS app which is great. However, I've removed the zone and it still appears in the app. In looking at HAAPI.swift, it appears that new zones are added and existing zones are updated. I don't see code that compares the zones from the server and removes the zones from the realm that don't exist in the server.

Might just be easiest to remove all the zones. Something like this in HAAPI.swift, storeEntities:

```
let realm = Current.realm()

let zones = realm.objects(RLMZone.self).map { $0 }

zones.forEach {
	realm.delete($0)
}
```
might do it.

**Comment by robbiet480** [comment]
Fixed in 2.0

**Comment by sgruby** [comment]
Thanks!
